package.json

```diff
@@ -59,10 +59,10 @@
     "bundle-stats": "npm run build:umd && webpack-bundle-analyzer dist/bundles/stats.json"
   },
   "dependencies": {
-    "@kontent-ai/core-sdk": "10.0.0",
+    "@kontent-ai/core-sdk": "^10.0.0",
```

Comment:
please remove the `^` as I prefer to use fixed versions to prevent possible issues. Quite recently there were bugs included in `axios` which would break this SDK if user installed it with `^`